lib/kennel/importer.rb

```diff
@@ -2,8 +2,15 @@
 
 module Kennel
   class Importer
+    # title will have the lock symbol we need to remove when re-importing
     TITLES = [:name, :title].freeze
-    SORT_ORDER = [*TITLES, :id, :kennel_id, :type, :tags, :query, *Models::Record.subclasses.map { |k| k::TRACKING_FIELDS }, :template_variables].freeze
```

Comment:
FYI this had an unrealted bug too where we needed flat_map because TRACKING_FIELDS can have multiple entries